Kyle_Marienthal/filter_by_type.py

```diff
@@ -0,0 +1,33 @@
+sI = 45
+mI = 100
+bI = 455
+eI = 0
+spI = -23
+sS = "Rubber baby buggy bumpers"
+mS = "Experience is simply the name we give our mistakes"
+bS = "Tell me and I forget. Teach me and I remember. Involve me and I learn."
+eS = ""
+aL = [1,7,4,21]
+mL = [3,5,7,34,3,2,113,65,8,89]
+lL = [4,34,22,68,9,13,3,5,7,9,2,12,45,923]
+eL = []
+spL = ['name','address','phone number','social security number']
+
```

Comment:
you may want to be more explicit with your variable names.  A good rule of thumb is, "If my co-worker came along and worked on my code would they understand my variable and function names?".